.golangci.yml

```diff
@@ -10,6 +10,9 @@ linters-settings:
     list-type: blacklist
     packages:
       - github.com/FerretDB/FerretDB/internal/bson
+      - github.com/FerretDB/FerretDB/internal/fjson
```

Comment:
I had several issues with trying to import tjson instead of fjson and vise-versa, so let's merge those rules now.

As for Windows, I think we should finish and merge #758 next week.